keps/sig-instrumentation/2831-kubelet-tracing/README.md

```diff
@@ -328,7 +305,17 @@ _This section must be completed when targeting beta graduation to a release._
   No impact to running workloads, logs will indicate the problem.
 
 ###### What specific metrics should inform a rollback?
-  To be determined.
+
```

Comment:
L296: Have those test been added?

FWIW - given that this isn't a persistent thing, rather in-memory switch, I would say that explicit enablement/disablement tests aren't really needed, as they won't provide visible value on top of the actual feature tests.

Reply:
there are no tests for enablement/disablement - i'll add a note there